sample/src/main/res/layout/fragment_radio_buttons.xml

```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.core.widget.NestedScrollView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content">
```

Comment:
Please add a 16dp padding to be aligned with the others sample screens :)

Reply:
Agree, I initially compared with the buttons layout and since there is no padding in this one, only 8dp margin on every component, I wasn't sure about the way to go. Maybe it requires a small refactoring.